### PR TITLE
Add hooks and services

### PR DESCRIPTION
**Pull request by noahjoeris**
This PR
- adds `useChains` and `usePastTransfers` hooks
- adds `chains` and `pastTransfers` services
- adds first draft of `transfer` model
- adds `useTokens` template that still needs to be implemented. As well as `tokens` service.

Note: To keep the PR smaller I will include `useOngoingTransfers`, `useTransfer` etc. in separat PRs.

### CODE REVIEW

File: app/src/services/pastTransfers.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Transfer } from '@/models/transfer'
+
+/**
+ * Fetches the finished transfers for a given address. Or for all addresses if no address is provided.
```
Comment (NunoAlexandre) [inline app/src/services/pastTransfers.ts:4]: Would this only be for "finished" transfers or also for transfers that are in progress?

Comment (noahjoeris) [reply app/src/services/pastTransfers.ts:4]: Only finished. The next PR will contain the `useOngoingTransfers` hook :)

File: app/src/components/NotificationToast.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const NotificationToast: React.FC<NotificationToastProps> = ({
   // Remove notification after TTL
   useEffect(() => {
     const timeoutRef = setTimeout(() => {
-      removeNotification(notification.id)
+      removeNotification(notification.id!)
```
[the remark this answers is not in the document]
Comment (noahjoeris) [reply app/src/components/NotificationToast.tsx:27]: Let me try to use the `Omit` utility type. This way we can get rid of the `!` I think.

[the remark this answers is not in the document]
Comment (noahjoeris) [reply app/src/components/NotificationToast.tsx]: Fixed ✅

**Review by NunoAlexandre — APPROVED** [review]
✨

**Pull request merged**